src/main/java/racinggame/view/ScoreScreen.java

```diff
@@ -0,0 +1,37 @@
+package racinggame.view;
+
+import java.util.List;
+import java.util.Map;
+import racinggame.domain.scoreboard.ScoreBoard;
+import racinggame.domain.scoreboard.Score;
+
+public class ScoreScreen {
+
+    public static void printWinner(ScoreBoard scoreBoard) {
+        List<String> winners = scoreBoard.getWinner(scoreBoard.getCountOfRound());
+        System.out.printf(
+                "%s가 최종 우승했습니다.%n",
+                String.join(" ,", winners.toArray(new String[0]))
```

Comment:
```suggestion
                String.join(", ", winners.toArray(new String[0]))
```

별건 아니지만, 이게 좀 더 자연스러워 보이네요 ㅎㅎ